backup/main.go

```diff
@@ -210,22 +196,16 @@ func doBackupRestore(c *cli.Context) error {
 	return nil
 }
 
-func doBackupRestoreIncrementally(c *cli.Context) error {
-	if c.NArg() == 0 {
+func DoBackupRestoreIncrementally(url string, deltaFile string, lastRestored string) error {
```

Comment:
This commit should be merged with the next commit since the next commit is the caller.